policyengine_us/tests/policy/baseline/gov/states/ny/tax/income/ny_supplemental_tax.yaml

```diff
@@ -277,3 +277,27 @@
     ny_main_income_tax: 11_963.755
   output:
     ny_supplemental_tax: 568
+
+- name: Unit test for year 2028 married filing jointly and qualifying widow(er)
+  absolute_error_margin: 0.1
+  period: 2028
+  input:
+    state_code: NY
+    filing_status: JOINT
+    ny_agi: 110_000 
+    ny_taxable_income: 27_900 
+    ny_main_income_tax: 1_202
+  output:
+    ny_supplemental_tax: 0
+
+- name: Unit test for 2028 separate filers
+  absolute_error_margin: 0.1
+  period: 2028
+  input:
+    state_code: NY
+    filing_status: SEPARATE 
+    ny_agi: 220_000
+    ny_taxable_income: 200_000
+    ny_main_income_tax: 11_963.755
+  output:
+    ny_supplemental_tax: 0
```

Comment:
but there is still a scenario where a person can have supplemental tax - just the incremental benefit was removed? - If so please add a test to show and adjust the parameter

Reply:
For the year 2028, it is impossible for `ny_agi > agi_limit` to return True, since the `agi_limit` is set to infinity according to the parameter file: https://github.com/PolicyEngine/policyengine-us/blob/f46557358a31de9b14d345b99c6be6d2591004a5/policyengine_us/parameters/gov/states/ny/tax/income/main/joint.yaml#L69

Reply:
Therefore, we can only return 0 for 
```
return where(
            ny_agi > agi_limit,
            supplemental_tax_high_agi,
            0,
        )
```